package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "dynamodb-capacity-manager",
+  "name": "@silvermine/dynamodb-capacity-manager",
```

Comment:
The README showing how to `require(...)` this library is actually already wrong. Could you fix it along with this PR?